study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/StudyBuilder.java

```diff
@@ -367,21 +368,43 @@ private void insertStudyGovernance(Handle handle, StudyDto studyDto) {
     private void insertStudyDetails(Handle handle, long studyId) {
```

Comment:
Wait. Hold on. Why are we updating this method? We should be changing the `insertStudySupportedLanguages()` method below.